9 implement groups

codeeMadness opened the pull request:
Hi,

This has follow your suggestion to implement peerjs-group.
However, I've just finished the implementation of chat group.
We will need other tickets to handle spawn players in group and call video in group.

So, please help to merge this first.
Many thanks.

codeeMadness committed:
9-implement-groups

-- add some materials support this feature

codeeMadness committed:
9-implement-groups

-- support group chats

codeeMadness committed:
change default model for classroom demo to make it lighter to load

Liquid-Blocks commented:
I have been experimenting a bit with this group feature and sadly do not think it is ready to be merged.

**Problem 1:** When creating a group, I am prompted with a message saying I need permission. 
![image](https://user-images.githubusercontent.com/46661277/171671970-9b4ad93a-6f8f-4230-8d7b-8209fb42b1d7.png)

**Problem 2:** There seems to be no ways of joining a group?
![image](https://user-images.githubusercontent.com/46661277/171673609-84ce5228-4263-48ec-b130-e3f5d0404f25.png)

There is no hurry, please take the time to complete issues properly so we don't accumulate cleanup jobs.

codeeMadness committed:
9-implement-groups

-- fix issue for administrator
-- add some indicator for clearer how to access group

codeeMadness commented:
Hi @Liquid-Blocks ,

I've just updated the code for cover above issues.

https://user-images.githubusercontent.com/102911684/171687868-716079d7-6247-4ada-bccf-27aed5035b45.mp4

Please take a look on this video, feel free to add some feedbacks, then I'm willing to update more.

Liquid-Blocks commented on js/util/RemoteController.js:
```diff
@@ -2,13 +2,24 @@ import { Vector3 } from 'https://cdn.skypack.dev/pin/three@v0.137.0-X5O2PK3x44y1
 const Peer = window.Peer;
+const SIGNALLING_OPTIONS =  {
+    secure: 'wss://',
+    host: 'spacetime-peerserver.herokuapp.com',
```
Hi @codeeMadness could you please open a new pr to include the server side code into the repository.

This repository is public, be careful not to include any private key or token.

thank you

Liquid-Blocks reviewed (approved):
Thank you for the update!
